core/src/main/java/ai/timefold/solver/core/api/score/buildin/bendablebigdecimal/BendableBigDecimalScore.java

```diff
@@ -46,22 +44,22 @@ public static BendableBigDecimalScore parseScore(String scoreString) {
      * Creates a new {@link BendableBigDecimalScore}.
      *
      * @param initScore see {@link Score#initScore()}
-     * @param hardScores never null, never change that array afterwards: it must be immutable
-     * @param softScores never null, never change that array afterwards: it must be immutable
-     * @return never null
+     * @param hardScores never change that array afterwards: it must be immutable
+     * @param softScores never change that array afterwards: it must be immutable
      */
-    public static BendableBigDecimalScore ofUninitialized(int initScore, BigDecimal[] hardScores, BigDecimal[] softScores) {
+    // TODO: @NonNull BigDecimal?
```

Comment:
Not sure what you mean. If the question is whether the array contents can be null, than the answer is no - but I'm not sure you can state that using these annotations.